data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -49,24 +49,29 @@ def test_transform(self, mock_etl):
         input_src = DATA_DIR / "input.csv"
         input_dst = etl.INPUT_CSV
         acs_src = DATA_DIR / "acs.csv"
+        # Lucas raised this as a lingering artifact. We have a
```

Comment:
Can you just delete `(acs_src, acs_dst)`? I think it'll all work fine to just do `for src, dst in [(input_src, input_dst)]:` since you no longer need that data file to be copied.